install/Knative-with-Gardener.md

```diff
@@ -148,11 +148,11 @@ spec:
 And of course create the respective `ConfigMaps`:
 
 ```
-curl https://github.com/knative/serving/releases/download/v0.2.3/istio.yaml
+curl https://github.com/knative/serving/releases/download/v0.3.0/istio.yaml
 kubectl create configmap istio-chart-080 --from-file=istio.yaml
 
-curl https://github.com/knative/serving/releases/download/v0.2.3/release.yaml
-kubectl create configmap knative-chart-001 --from-file=release.yaml
+curl https://github.com/knative/serving/releases/download/v0.3.0/serving.yaml
```

Comment:
Should we now create a configmap for both Knative Serving and Knative Build? Right now, it only creates one for Serving. (Originally it was creating a configmap for the bundled release, which had Serving + Build.)

Reply:
Thanks for housekeeping.
We are re-working the Bouquet addon manager completely and have deprecated that feature for now.
/cc @adracus
The section `## Alternative way to enable Knative with Gardener` can be removed for the time being.

I think there already was a comment that the install information is mostly redundant and could be references from one place. The only the valuable info remaining is how to get the initial cluster and what steps are necessary to configure the underlay properly to get istio/knative working orderly. If you are reworking the doc to that target, this Gardener install doc can be minimized further.

Reply:
Thanks!